examples/with-okta-multi-tenant-pkce-flow/src/routes/index.tsx

```diff
@@ -0,0 +1,27 @@
+import {useState} from 'react';
+
+export default function Index() {
+
+  const [tenant, setTenant] = useState('');
+
+  function handleClick(e: any) {
+    e.preventDefault();
+    window.localStorage.setItem('tenant', tenant);
```

Comment:
Explain why this is being stored in localstorage

Reply:
done